app/queries/crash.ts

```diff
@@ -235,3 +235,17 @@ export const CREATE_CRASH = gql`
     }
   }
 `;
+
+export const DELETE_CRIS_CRASH = gql`
+  mutation DeleteCrisCrash($id: Int!) {
+    update_crashes_cris(
+      where: { id: { _eq: $id }, is_temp_record: { _eq: true } }
```

Comment:
using `is_temp_record: { _eq: true }` ensures that we cannot soft-delete anything but temp records 😅